src/pages/MonthlyGood/FrequentlyAskedQuestions.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+	<div class="frequently-asked-questions-section-wrapper">
```

Comment:
This wrapping div is unnecessary and the `.row` div can be used as the root element instead.